sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3496,6 +3496,88 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
     checkIfSeedExistsInExplain(df2)
   }
 
+  test("SPARK-31670: Struct Field in groupByExpr with CUBE") {
```

Comment:
Could you please make the title more correct? I think we don't need the word `with CUBE`.

Reply:
> Could you please make the title more correct? I think we don't need the word `with CUBE`.

It's ok for current PR title?

Reply:
Yea, looks fine.